tinymovr_joint_iface:
  joints:
    wheel_2:
      # hardware ID of the actuator
      id: 2
      offset: 3.14159265359 # in rad
      delay_us: 200
      #max-speed: 4.0 # in rad/s
      rads_to_ticks: 14.323944878 # how many ticks in a rad
      command_interface: velocity
    wheel_1:
      # hardware ID of the actuator
      id: 1
      offset: 3.14159265359 # in rad
      delay_us: 200
      #max-speed: 4.0 # in rad/s
      rads_to_ticks: 14.323944878 # how many ticks in a rad
      command_interface: velocity

